NestedAccordion/js/scripts.js

```diff
@@ -0,0 +1,49 @@
+/* SO after inspecting all the different elements inside the hmtl and seeing how
```

Comment:
like we saw in class, it's good to break pseudocode down into comments, one line per comment.